packages/fluentui/docs-components/src/ComponentControlsCodeSandbox/types.tsx

```diff
@@ -0,0 +1,7 @@
+export type CodeSandboxImport = {
+  module: any;
+  version: string;
+  required: boolean;
+};
+
+export type SourceManagerLanguage = 'js' | 'ts';
```

Comment:
```suggestion
export type CodeSandboxLanguage = 'js' | 'ts';
```

Nit: as now it's a type for `CodeSandbox` component